lib/src/cmcd/CMCD_HEADERS.ts

```diff
@@ -0,0 +1 @@
+export const CMCD_HEADERS = 'headers' as const;
```

Comment:
Given you have a string literal in const variables, you don't need `as const`, you will get string literal type for free

Example on the playground - https://tsplay.dev/WyZZZw

Unfortunately, it doesn't work for let variables, so if you'd like to achieve a consistency in the codebase, this is a good approach then.

Have you added `as const` for consistency or are there other reasons that I'm missing?

Reply:
@littlespex right, given `isolatedDeclaration ` is enabled in tsconfig, when you don't have `as const` on the const variable, would it be resolved as a `string`?

Reply:
At some point during the refactor adding `as const` to exports fixed compile errors I was seeing, but it looks like they are no longer needed. They have been removed from the exports. Thanks for the feedback.